Automattic/vip-go-mu-plugins
# Search: Health - validate counts: prevent potential fatal for term and comment indexables.

## Description

This is a hotfix for an issue introduced in #2384 which switched from hard coded user and post counts to all active indexables. 

Unfortunately, other indexables are not handled properly in `validate_indexable_count_for_site` method, resulting in 
`null` being passed to `render_results` which expects an array, thus causing an `Uncaught Type Error` fatal.

props @dhusakovic for the find

## Changelog Description

### Plugin Updated: Enterprise Search

We've applied a hotfix to address a potential fatal error when running `wp vip-search health validate-counts` commands for Term and Comment indexables.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Run the command on a site with Term or Comment Indexable enabled.
1. Verify there's no fatal anymore
